cloud/aws/templates/aws_oidc/alarms.tf

```diff
@@ -34,6 +34,25 @@ resource "aws_cloudwatch_metric_alarm" "cpu_utilization_too_high" {
   }
 }
 
+// Memory Utilization
+resource "aws_cloudwatch_metric_alarm" "memory_utilization_too_high" {
+  count               = var.rds_create_high_memory_alarm ? 1 : 0
+  alarm_name          = "rds-${data.aws_db_instance.civiform.id}-highMemoryUtilization"
```

Comment:
**suggestion:** Sorry for not noticing this yesterday, but currently this is set up for RDS, but we'll want it for ECS instead. This will involve updating any RDS references in the variable names and the alert values to ECS.

Reply:
ok! should I also add a cpu utilization alarm for ecs?

Reply:
We actually have one setup as part of the [ECS autoscaling policy](https://github.com/civiform/cloud-deploy-infra/blob/40c339725e080607a22708279e320ce656ff3cab/cloud/aws/modules/ecs_fargate_service/main.tf#L297). It currently is set up as part of another [terraform module](https://github.com/cn-terraform/terraform-aws-ecs-service-autoscaling), but we have another issue to remove that dependency, so its more obvious that the alarm exists https://github.com/civiform/civiform/issues/5632. So don't worry about adding another alarm for ECS CPUUtilization, but thank you for thinking of that!

Reply:
Should I:
* generalize the actions email so it isn't rds_alarm_email
* create a new ecs actions email and sns topic?

Reply:
Good question! Let's generalize, but Arkansas is using the rds_alarm_email variable already, so we'll have to let them know to update that when they run the next deployment.